### ISSUE TITLE
Change to a flat Windows logo 

### ISSUE THREAD

**Issue by djsvi** [amidaware/tacticalrmm#1010]
I'd like the Windows logo image to be swapped with a flat version in keeping with the rest of TRMM's styling.

I'd much prefer a colourful one - https://commons.wikimedia.org/wiki/File:Microsoft_logo.svg - but if it must be blue there's this one - https://commons.wikimedia.org/wiki/File:Windows_logo_-_2021.svg

Thanks!

**Comment by sadnub** [comment]
These are the icons available to choose from: https://quasar.dev/vue-components/icon#introduction

At the top of that page it shows the different icon services. Can you find something in there?

**Comment by djsvi** [comment]
Ah. How about <font-awesome-icon icon="fa-brands fa-microsoft" /> then?

**Comment by djsvi** [comment]
As a test I used the developer console to change from mdi-microsoft-windows to just **mdi-microsoft** and I think it looks much nicer flat..

![image](https://user-images.githubusercontent.com/79456430/159370483-861a5191-9250-45ee-a2c9-0a316689297e.png)

**Comment by silversword411** [comment]
The all-square doesn't match Win10 start button, which the current icon does. 

![2022-03-21_220802 - explorer_Program_Manager](https://user-images.githubusercontent.com/13395348/159393127-f8fe540c-3b4d-45cd-86ae-97d6abff9a26.png)

**Comment by djsvi** [comment]
> The all-square doesn't match Win10 start button, which the current icon does. #JustSayin
> 
> ![2022-03-21_220802 - explorer_Program_Manager](https://user-images.githubusercontent.com/13395348/159393127-f8fe540c-3b4d-45cd-86ae-97d6abff9a26.png)

Yep, it looks crap there too :) Microsoft replaced it with a flat version in Windows 11.

**Comment by silversword411** [comment]
This is a personal taste...I think you'd need to have a vote on discord to decide this one ;)

**Comment by wh1te909** [comment]
we should add a logos override file where you can specify your own logos for certain things like this

**Comment by plao** [comment]
> we should add a logos override file where you can specify your own logos for certain things like this

👍

**Comment by sadnub** [comment]
Lets do a vote. I personally like the non-flat icon lol.

**Comment by dinger1986** [comment]
I'm with you @sadnub much prefer the non flat one

**Comment by djsvi** [comment]
I vote for the flat icon. 

Here's a Tampermonkey script for those who prefer the flat icon (assuming the majority vote is to keep the current icon).

```
// ==UserScript==
// @name         Flat Microsoft Icon
// @namespace    http://tampermonkey.net/
// @version      0.1
// @description  replace mdi-microsoft-windows with mdi-microsoft 
// @author       You
// @match        https://yourtacticalrmmURL/
// ==/UserScript==

(function() {
    document.querySelectorAll(".mdi-microsoft-windows").forEach(elem => {
        elem.classList.remove("mdi-microsoft-windows");
        elem.classList.add("mdi-microsoft");
    });

    setTimeout(arguments.callee, 300);
})();  
```
